tests/all_tests.sh

```diff
@@ -1,13 +1,9 @@
 #!/bin/bash
 
-srcdir="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. && pwd)"
+cd "$(dirname "$0")"/..
+jobs=${JOBS:-4}
+filter=${FILTER:-""}
 
-test_status=0
-test_files=( "$srcdir"/tests/tokenized/*.txt )
-for tf in "${test_files[@]}"; do
-    test_name="$(basename "$tf")"
-    test_name="${test_name%.txt}"
-    "$srcdir"/tests/decompyle_test.sh $test_name tests || test_status=1
-done
-
-exit $test_status
+find tests/tokenized -type f -name '*.txt' -a -name "*${filter}*" -print0 | \
```

Comment:
Minor nitpick, but it would be nice to only add the `-a -name "*${filter}*"` parameters when FILTER is specified.

Reply:
I think it adds too much complexity when this one-line works. When FILTER is not defined or empty, the globing will become `**` which will match everything. My goal with this PR was to add simple parallelism without trying to re-implement pytest-xdist in bash ^^.